app/src/main/java/ar/scacchipa/twittercloneapp/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package ar.scacchipa.twittercloneapp.viewmodel
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class LoginViewModel: ViewModel() {
+
+    private val _mustShowErrorMsg = MutableLiveData(false)
+    private val _navToFragAuthWeb = MutableLiveData(false)
+
+    fun showErrorMsg() {
+        _mustShowErrorMsg.value = true
+    }
+
+    fun onClickMsgButton() {
+        _mustShowErrorMsg.value = false
+    }
+
+    fun addMsgChangeObserver( owner: LifecycleOwner, observer: (Boolean) -> Unit) {
```

Comment:
I dont think is useful, it brings more problems when testing than solutions.

Reply:
It is unnecessary and It was removed